app/jobs/process_cdr_job.rb

```diff
@@ -23,12 +35,13 @@ def perform(cdr)
         content_type: "application/json"
       }
     )
-
-    update_phone_call_status(phone_call)
-    notify_status_callback_url(phone_call)
   end
 
-  private
+  def call_leg_A?(cdr)
```

Comment:
Maybe this is easier to read if we change this method to return true if it's leg_b instead?

It's a bit hard to understand but if we find something that identifies it as call_leg_b such as the presence of the originator or bridge_uuid it might be easier to understand.

Reply:
Either way should be the same, but I'm okay if the leg_b is easier to read than leg_a.

Reply:
I tried to understand it but I couldn't. Also it's seems safer that it falls back to leg A if the condition is not met